README.md

```diff
@@ -17,14 +17,16 @@ environment for building Zephyr apps, follow the [Getting Started guide](https:/
 
 Then you can setup your workspace to build this application by running:
 
-```
-west init -m https://github.com/golioth/golioth-memfault-zephyr-example.git
+```bash
+mkdir memfault-golioth-example
+cd memfault-golioth-example
+west init -m https://github.com/memfault/memfault-golioth-example.git
```

Comment:
LMK what you'd like to do with the URL @sam-golioth , unfortunately I didn't come across an easy way to make it dynamic on the GitHub repo